uniplot/pixel_matrix.py

```diff
@@ -18,6 +18,8 @@ def render(
     width: int,
     height: int,
     lines: bool = False,
+    pixels: NDArray | None = None,
```

Comment:
changes here are backwards compatible, so nothing really changed, we just support new things for later optimizations